src/data/queries/audienceTopics.ts

```diff
@@ -12,6 +11,18 @@ export const params: QueryParams<null> = () => {
   ])
 }
 
+const formatBeneficiariesData = (beneficiaries) => {
```

Comment:
This file was missing logic to handle a single object of beneficiaries rather than an array, so I added a formatter function instead.